Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.yml

```
@@ -63,7 +63,8 @@ configuration:
   required: false
   type: 9
 - additionalinfo: Indicators will be fetched from this collection. Run "taxii2-get-collections"
-    command to get a valid value.
+    command to get a valid value. If left empty, the instance will try to fetch from
+    all indicators.
```

not sure mine is good either
```suggestion
    all the collections in the given discovery service.
```